tests/saw/spec/DRBG/DRBG.saw

```diff
@@ -99,6 +99,8 @@ let alloc_blob n = do {
     datap <- alloc_bytes n;
     crucible_points_to (crucible_field p "data") datap;
     crucible_points_to (crucible_field p "size") (tm {{ `n : [32] }});
+    crucible_points_to (crucible_field p "allocated") (tm {{ 0 : [32] }});
+    crucible_points_to (crucible_field p "growable") (tm {{ 0 : [8] }});
```

Comment:
@jldodds approved?

Reply:
Looks great. It's worth noting that this isn't a "full" proof for the function, but it is sufficient for the use by DRBG.